api/admin/routes.py

```diff
@@ -288,6 +288,18 @@ def libraries():
         return data
     return flask.jsonify(**data)
 
+@app.route("/admin/library/<library_uuid>", methods=["DELETE"])
+@returns_problem_detail
+@requires_admin
+@requires_csrf_token
+def library(library_uuid):
+    data = app.manager.admin_settings_controller.library(library_uuid)
+    if isinstance(data, ProblemDetail):
+        return data
+    if isinstance(data, Response):
+        return data
+    return flask.jsonify(**data)
+
```

Comment:
There are so many copies of this code -- do a thing, check what was returned, jsonify it -- that it might be worth creating yet another decorator and putting the code in there.

Reply:
Done.

Reply:
This looks great!